Terraform/GCP/k8s/values/elastic.yaml

```diff
@@ -217,29 +204,256 @@ vector:
         compression: none
         endpoints:
           - "${ES_ENDPOINT}"
-        id_key: id
         mode: bulk
         bulk:
-          action: create
-          index: vector-logs-syslog-%Y-%m-%d
+          action: index
+          index: vector-logs-syslog-%F
         auth:
           strategy: basic
           user: "${ES_USERNAME}"
           password: "${ES_PASSWORD}"
-      k3s_logs:
+      es_logs:
         type: elasticsearch
         inputs:
-          - k8s_all
+          - k3s_logs
         api_version: v7
         compression: none
         endpoints:
           - "${ES_ENDPOINT}"
-        id_key: id
         mode: bulk
+        id_key: _id
         bulk:
-          action: create
-          index: vector-logs-k3s-{{`{{ kubernetes.pod_name }}`}}-%Y-%m-%d
+          action: index
+          index: vector-logs-kubernetes-%F
         auth:
           strategy: basic
           user: "${ES_USERNAME}"
           password: "${ES_PASSWORD}"
+
+metricbeat:
```

Comment:
need to change this to
```yaml
metricbeat:
  enabled: true
  daemonset:
    # Include the daemonset
    enabled: true
  # other config
```

to be able to disable/enable this subchart

Reply:
fixed